app/controllers/v0/appeals_controller.rb

```diff
@@ -2,6 +2,8 @@
 
 module V0
   class AppealsController < AppealsBaseController
+    service_tag 'appeal-status'
```

Comment:
This controller is used in the same application as the others here (Claim Status Tool). It's still quite possible that `appeal-status` is the correct tag though, just wanted to point that out